US County Data - pandas.ipynb

```diff
@@ -7,7 +7,8 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    top_states.head()
</pre><p>Again, this is really not the "Pandas" way of doing things, you should just write a filter and set it equal to a new dataframe.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CMU-MS-DAS-Modern-Programming-Mini/ClassAssignmentNo4/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>